portfolio/src/main/webapp/style.css

```diff
@@ -24,7 +24,7 @@
 html{
     font-size: 1.5rem;
     font-family: 'arial', sans-serif;
-    color:rgb(255, 254, 254);
```

Comment:
Any reason for the change here and below? I think its generally preferable to use the hex codes.

Reply:
My bad, I think that was a mistake! I meant to change them all to color:white!